src/Backend/Core/Js/ckeditor/plugins/medialibrary/dialogs/linkDialog.js

```diff
@@ -208,8 +208,9 @@ CKEDITOR.dialog.add(
                                     url.searchParams.append('body', this.getDialog().getValueOf('tab', 'content'));
                                 }
 
-                                element.setAttribute('href', url.toString());
-                                element.setAttribute('data-cke-saved-href', url.toString());
+                                var href = url.toString().replace(/\+/g, '%20');
```

Comment:
Can't this have problems when e.g. query parameters contain a plus? Not sure what's exactly possible here but this seems quite risky 🤔

Reply:
```
var url = new URL('https://www.fork-cms.com')
url.searchParams.append('foo', 'bar + bar = 20')
console.log(url.toString()) // https://www.fork-cms.com/?foo=bar+%2B+bar+%3D+20
console.log(url.toString().replace(/\+/g, '%20')) // https://www.fork-cms.com/?foo=bar%20%2B%20bar%20%3D%2020
```

So it shouldn't cause any issues